src/scripts/features/registrar_nft_transfer.feature

```diff
@@ -0,0 +1,145 @@
+@registrar
+@dev
+Feature: Yumi market transfer Api
+
+  Background:
+    Given Reinstall registrar related canisters
+    Then Admin import names as following
+      | name      | owner | years |
+      | name1.ic  | user1 | 1     |
+      | na2.ic    | user1 | 2     |
+      | iiiiii.ic | user2 | 3     |
+
+  Scenario: Ext transfer success
+    Then registrar get_details "name1.ic" result is
+      | key        | value    |
+      | owner      | user1    |
+      | name       | name1.ic |
+      | expired_at | 1        |
+      | created_at | 0        |
+    Given registrar ext_transfer action
+      | caller | name     | from  | to    | from_type | to_type   |
+      | user1  | name1.ic | user1 | user2 | principal | principal |
+    When all registrar ext_transfer is ok
+    And registrar get_details "name1.ic" result is
+      | key        | value    |
+      | owner      | user2    |
+      | name       | name1.ic |
+      | expired_at | 1        |
+      | created_at | 0        |
+
+  Scenario: Ext transfer from the allowed caller success,The parameter to user is ignored
+    Then registrar get_details "name1.ic" result is
+      | key        | value    |
+      | owner      | user1    |
+      | name       | name1.ic |
+      | expired_at | 1        |
+      | created_at | 0        |
+    Given registrar ext_approve action
+      | spender | name     |
+      | user3   | name1.ic |
+    Given registrar ext_transfer action
+      | caller | name     | from  | to    | from_type | to_type   |
+      | user3  | name1.ic | user1 | user2 | principal | principal |
+    When all registrar ext_transfer is ok
+    Then registrar get_details "name1.ic" result is
+      | key        | value    |
+      | owner      | user3    |
```

Comment:
if user1 approve to user3, then user3 can transfer from user1 to user2 in this case.
However, this is wrong